MusicXML.scm

```diff
@@ -266,8 +267,8 @@
 
       ((NoteEvent)
        (let* ((pitch (ly:music-property m 'pitch))
-              (tie-start (if tie-start-pitches (in-list? pitch tie-start-pitches) #f))
-              (tie-stop (if tie-stop-pitches (in-list? pitch tie-stop-pitches) #f)))
+              (tie-start (if tie-start-pitches (lset<= equal? (list pitch) tie-start-pitches) #f))
+              (tie-stop (if tie-stop-pitches (lset<= equal? (list pitch) tie-stop-pitches) #f)))
```

Comment:
Here I think it would work and be simpler to use Guile's `member` function:
See: https://www.gnu.org/software/guile/manual/guile.html#List-Searching

Reply:
Yes, thanks!  I actually need a boolean here, and `member` might return a list, but I guess it will work as a boolean #t downstram anyway.

Reply:
Got it down to a simpler form. @PaulMorris see if this looks good now.